src/index.js

```diff
@@ -240,50 +241,91 @@ export default class SauceLabs {
       }
     }
 
-    let sauceConnectVersion = argv.scVersion;
-    if (!sauceConnectVersion) {
-      sauceConnectVersion = await this._getLatestSauceConnectVersion();
+    const sauceConnectVersion = argv.scVersion || DEFAULT_SAUCE_CONNECT_VERSION;
+    if (sauceConnectVersion.startsWith('4')) {
+      throw new Error(
+        `This Sauce Connect version (${sauceConnectVersion}) is no longer supported. Please use Sauce Connect 5.`
+      );
     }
+
+    const scUpstreamProxy = argv.scUpstreamProxy;
     const args = Object.entries(argv)
       /**
        * filter out yargs, yargs params and custom parameters
        */
       .filter(
         ([k]) =>
-          !['_', '$0', 'sc-version', 'logger', ...SC_PARAMS_TO_STRIP].includes(
-            k
-          )
+          ![
+            '_',
+            '$0',
+            'sc-version',
+            'sc-upstream-proxy',
+            'tunnel-name',
+            'logger',
```

Comment:
How about just simply extending the `SC_PARAMS_TO_STRIP` with those additional keys?